fluentd/rh-manifest.txt

```diff
@@ -1,83 +1,65 @@
-activemodel 5.2.4.2 http://rubyonrails.org
-activesupport 5.2.4.2 http://rubyonrails.org
-addressable 2.7.0 https://github.com/sporkmonger/addressable
-aes_key_wrap 1.0.1 https://github.com/tomdalling/aes_key_wrap
-attr_required 1.0.1 http://github.com/nov/attr_required
-bindata 2.4.6 http://github.com/dmendel/bindata
-concurrent-ruby 1.1.6 http://www.concurrent-ruby.com
-connection_pool 2.2.2 https://github.com/mperham/connection_pool
-cool.io 1.6.0 http://coolio.github.com
-digest-crc 0.5.1 https://github.com/postmodern/digest-crc#readme
-domain_name 0.5.20190701 https://github.com/knu/ruby-domain_name
-elasticsearch 7.5.0 http://github.com/elasticsearch/elasticsearch-ruby
-elasticsearch-api 7.5.0 https://github.com/elasticsearch/elasticsearch-ruby/tree/master/elasticsearch-api
-elasticsearch-transport 7.5.0 https://github.com/elasticsearch/elasticsearch-ruby/tree/master/elasticsearch-transport
-excon 0.73.0 https://github.com/excon/excon
-faraday 0.17.3 https://lostisland.github.io/faraday
-ffi 1.12.2 https://github.com/ffi/ffi/wiki
+digest-crc 0.6.1 https://github.com/postmodern/digest-crc#readme
+dig_rb 1.0.1 https://github.com/jrochkind/dig_rb
 ffi-compiler 1.0.1 http://wiki.github.com/ffi/ffi
 fluent-config-regexp-type 1.0.0 https://github.com/okkez/fluent-config-regexp-type
-fluentd 1.9.2 https://www.fluentd.org/
-fluent-mixin-config-placeholders 0.4.0 https://github.com/tagomoris/fluent-mixin-config-placeholders
+fluentd 1.7.4 https://www.fluentd.org/
```

Comment:
Why does the script think we use 1.7.4 here?

Reply:
updated to truly be 1.11.1